scripts/benchmark/resvg.js

```diff
@@ -6,13 +6,9 @@ export default svg => (
 			loadSystemFonts: false,
 		},
 		logLevel: 'off',
-		imageRendering: 1,
-		textRendering: 0,
-		shapeRendering: 0,
```

Comment:
I was hoping these values would help speed up the process.

Reply:
This will change the rendering result of the PNG